src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -437,13 +456,121 @@ internal PackageManagerSearchViewModel()
             SelectedHosts = new List<string>();
         }
 
+        /// <summary>
+        /// Initialize Lucene config file writer.
+        /// </summary>
+        private void InitializeLuceneConfig()
```

Comment:
Hi @reddyashish A lot of these looks like shared code between package manager and other place. Do you think we can move these code to a shared class to cross reference?, e.g. LuceneSearchModel etc

Reply:
Yes, just the directory path is changed in the InitializeLuceneConfig() and the fields are different for InitializeIndexDocument().

Reply:
Gotcha, if only the path is different, we can consider passing path or sub folder name as param so we can share these code somehow